docs/usage.rst

```diff
@@ -12,6 +12,12 @@ options. The following sections will describe each in detail.
 General
 -------
 
+Note that |project_name| will ask you for your repo's passphrase
+which blocks the import until you enter it. To let |project_name|
+continue automatically, you can pass the environmental variable
```

Comment:
**environment** variable

Reply:
also, it is `borg` asking for the passphrase, right? not `borg-import`.

Reply:
> environment variable

Done, thanks!

> also, it is borg asking for the passphrase, right? not borg-import.

This might be right technically, but to me as a user it seems like borg-import asked for it, since I executed a borg-import command and not a borg one.
However, I can change it from borg-import to borg if you want. I just think that it might confuse people not knowing the relationship between borg and borg-import.